### DISCUSSION TITLE
Add new field, key, or value on auth

### DISCUSSION THREAD

**Discussion by ithustle** [invertase/react-native-firebase#5408] [Q&A] [answered]
Hi,
Probably duplicate but, I'd like to know if, is it possible add a new key on current user signed?

e.g

````
import  firebase from 'react-native-firebase/app';
import 'react-native-firebase/auth';
...
const user firebase.auth().currentUser;

...
user.newKey = 'Some value';

````

**Comment by mikehardy** [comment, marked as answer]
That object is typed, and has type definitions. It sound like you want a composite object that contains the user plus your information? With javascript (not typescript) you can do whatever you like in your code, but it won't be saved in the auth store.

**Reply by ithustle** [reply]
Okay. Thanks you.
